admissions/templates/admissions/index.html

```diff
@@ -56,6 +57,59 @@
                         </div>
                     </div>
                 </div>
+                {% elif app.application_status == "accepted" %}
+                <div class="card"
+                     style="margin-top: 3em; margin-left: 17em; margin-right: 17em; background-color: #F9F7F3">
```

Comment:
this code is repetitive. Use if-else only for the code which changes.

Reply:
> this code is repetitive. Use if-else only for the code which changes.

the color of each card is different , so its not repetititve

Reply:
You can use django variable for that

Reply:
> You can use django variable for that

Now not only color, but also status in header. Lets keep it this way